addons/l10n_in_tcs_tds/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_in_pan = fields.Char(string="PAN")
```

Comment:
I think the comment in the previous PR from @qdp-odoo still stands.  (we should have a help message at least to know why we have this field.

Reply:
okay, @hapa-odoo please add a help message
"A permanent account number is a ten-character alphanumeric identifier, by the Indian Income Tax Department"